src/anndata/_settings.py

```diff
@@ -109,6 +151,7 @@ def describe(
         option: str | Iterable[str] | None = None,
         *,
         print_description: bool = True,
```

Comment:
While we're at it, a ctrl+f for:
```suggestion
        should_print_description: bool = True,
```
would be better IMO